### PR TITLE
Set api-gateway pipeline to mandatory

### PR DESCRIPTION
**Pull request by barchw**
**Description**

Changes proposed in this pull request:

- Set api-gateway pipeline to mandatory as it is stable

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -24,10 +24,10 @@ plank:
...
         censoring_buffer_size: 3092 # 3kB
       utility_images:
-        clonerefs: gcr.io/k8s-prow/clonerefs:v20220721-cf42b99a12
-        initupload: gcr.io/k8s-prow/initupload:v20220721-cf42b99a12
-        entrypoint: gcr.io/k8s-prow/entrypoint:v20220721-cf42b99a12
-        sidecar: gcr.io/k8s-prow/sidecar:v20220721-cf42b99a12
+        clonerefs: gcr.io/k8s-prow/clonerefs:v20220822-46c56dcda7
+        initupload: gcr.io/k8s-prow/initupload:v20220822-46c56dcda7
+        entrypoint: gcr.io/k8s-prow/entrypoint:v20220822-46c56dcda7
+        sidecar: gcr.io/k8s-prow/sidecar:v20220822-46c56dcda7
```
Comment (Ressetkk) [inline prow/config.yaml:30]: Why is there this change?

Comment (barchw) [reply prow/config.yaml:30]: I was wondering myself, it was autogenerated by running:
go run development/tools/cmd/rendertemplates/main.go --data templates/data

Comment (barchw) [reply prow/config.yaml:30]: Should I revert this?

Comment (barchw) [reply prow/config.yaml]: OK I reverted those

**Pull request merged**